libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/audio/AudioTypeResolver.kt

```diff
@@ -0,0 +1,72 @@
+package com.mapbox.navigation.core.telemetry.audio
+
+import android.content.Context
+import android.media.AudioManager
+import android.os.Build
+
+internal sealed class AudioTypeResolver {
```

Comment:
Same as above re: move to `libnavigation-util`.

Reply:
👍